CHANGELOG.md

```diff
@@ -63,12 +79,6 @@ Pylance's copy of Pyright has been updated from 1.1.348 to 1.1.350.
 
 -   See Pyright's release notes for details: [1.1.349](https://github.com/microsoft/pyright/releases/tag/1.1.349), [1.1.350](https://github.com/microsoft/pyright/releases/tag/1.1.350)
 
-## 2024.2.3 (4 March 2024) Release
```

Comment:
Why did this one disappear?

Reply:
something wrong with the script, let me fix it.

Reply:
oh because the pyrx repo doesn't have it